src/ark/phenotyping/cluster_helpers.py

```diff
@@ -58,17 +58,19 @@ def normalize_data(self) -> pd.DataFrame:
                 The data with `columns` normalized by the values in `norm_data`
         """
 
-    def train_som(self, data: pd.DataFrame):
+    def train_som(self, data: pd.DataFrame, seed=42):
         """Trains the SOM on the data provided and saves the weights generated
 
         Args:
             data (pandas.DataFrame):
                 The input data to train the SOM on.
+            seed (int):
+                The random seed to use.
         """
-        # make sure to run a deterministic SOM for reproducibility purposes
+
         som_weights = som(
             data=data.values, xdim=self.xdim, ydim=self.ydim, rlen=self.num_passes,
-            alpha_range=(self.lr_start, self.lr_end), deterministic=True
+            alpha_range=(self.lr_start, self.lr_end)
```

Comment:
I think `seed` should be fed into the `som` function?

Reply:
Ah yes, long day today.